code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -93,6 +93,8 @@
 
 /datum/reagent/medicine/tramadol/on_mob_life(mob/living/L)
 	L.reagent_pain_modifier += PAIN_REDUCTION_VERY_HEAVY
+	if(volume > 5)
+		L.adjust_drugginess(1.1)
```

Comment:
The effect of this is a meme. Test it locally.

Reply:
I have. You get a minor druggy screen if you take a pill, there's no actual effect gameplay-wise. I'll take a look to see if your new changes make it get worse over time, though.